Content.Server/Xenoarchaeology/Equipment/Systems/NodeScannerSystem.cs

```diff
@@ -15,6 +16,34 @@ public sealed class NodeScannerSystem : EntitySystem
     public override void Initialize()
     {
         SubscribeLocalEvent<NodeScannerComponent, AfterInteractEvent>(OnAfterInteract);
+        SubscribeLocalEvent<NodeScannerComponent, GetVerbsEvent<UtilityVerb>>(AddUiVerb);
+    }
+
+    private void AddUiVerb(EntityUid uid, NodeScannerComponent component, GetVerbsEvent<UtilityVerb> args)
```

Comment:
Why Add*Ui*Verb? AddScanVerb instead?

Reply:
Started by copypasting that function, and I glossed over the name with the thought "This name sounds quite general, it's probably just convention or something"
I guess I'll fix this at some point